### DISCUSSION TITLE
Forcing display of cursor

### DISCUSSION THREAD

**Discussion by karim-ammouche** [FreeRDP/FreeRDP#9275] [Q&A]
Hello,
When a Qt application hide its cursor using setOverrideCursor(Qt::BlankCursor), it is not visible on xfreerdp client (tested with version 2.20 on Ubuntu 20.04)
The behavior is different with Windows wfreerdp where the native cursor is displayed.

Is there a way to force displaying the cursor on xfreerdp client ?

Thanks,
Best regards

**Comment by akallabeth** [comment]
no, you should fix your `qt` application as that should reset the cursor if not in its own application window.
you can, of course, start `xfreerdp` on a separate `X11` instance, but don´t know how to configure `Xwayland` for that. (not possible with plain `X11` sessions except letting it run in a `Xephr` window or similar)
